homeassistant/components/homeassistant/strings.json

```diff
@@ -46,6 +46,10 @@
           }
         }
       }
+    },
+    "config_entry_reauth": {
+      "title": "[%key:common::config_flow::title::reauth%]",
+      "description": "Reauthentication is needed"
```

Comment:
These will need to be adjusted or removed when we know how frontend will display these issues.